kolibri/core/urls.py

```diff
@@ -64,6 +65,7 @@
     url(r"^api/", include("kolibri.core.api_urls")),
     url(r"", include(i18n_patterns(lang_prefixed_patterns))),
     url(r"", include("kolibri.core.content.urls")),
+    url(r"^health/", HealthCheckView.as_view(), name="health_check"),
```

Comment:
Maybe..

```suggestion
    url(r"^status/", StatusCheckView.as_view(), name="status_check"),
```

we've had a need for something like this in windows, mac, endless, etc.